File: Niveau 1/7 – Conditions avancées, opérateurs booléens/02 - Maison de l'espion.py
minX = int(input())
maxX = int(input())
minY = int(input())
maxY = int(input())
maisonSuspecte = 0
for i in range(int(input())):
    X = int(input())
    Y = int(input())
    if minX <= X <= maxX and minY <= Y <= maxY:
        maisonSuspecte += 1
print(maisonSuspecte)

File: Niveau 1/7 – Conditions avancées, opérateurs booléens/02 - Maison de l'espion.cpp
#include <iostream>
using namespace std;
int main()
{
  int minX, maxX, minY, maxY, nbMaison, n = 0;
  cin >> minX >> maxX >> minY >> maxY >> nbMaison;
  for (int i = 0; i < nbMaison; i++)
  {
    int x, y;
    cin >> x >> y;
    if (minX <= x && maxX >= x && minY <= y && maxY >= y)
    {
      n++;
    }
  }
  cout << n;
}